Sender/data_from_csv.cpp

```diff
@@ -0,0 +1,53 @@
+#include"data_from_csv.h"
+
+DataFrom_csv::DataFrom_csv(std::string& fileName): _csvReader(fileName)
+{	if (_csvReader.ReadIsGood)
+	{	_file_lines = _csvReader.GetFileLines();
+		FormatLines_csv(); 
+	}
+}
+void DataFrom_csv::FormatLines_csv()	
+//needs to be changed if input file format is different
+{	UpdateOutputFormat();
+	UpdateMonthAndYear();
+	UpdateFootfallValues();
+}
+void DataFrom_csv::UpdateOutputFormat()
+{	const char* outputFormat = _file_lines[0].c_str();
```

Comment:
defensive check if `_file_lines` is not empty. We can remove the local variable `outputFormat`, typecast and pass the data to the `UpdateFormat` function.

Reply:
yes, done.